Documentation/alternate-fetches.md

```diff
@@ -0,0 +1,85 @@
+</script>                                                                       
+</head><body>                                                                   
+    <button id="login">go</button>                                              
+</body>                                                                         
+<script>                                                                        
+    const idp = "https://solidcommunity.net";                                   
+    const privateResource = "https://jeff-zucker.solidcommunity.net/private/";  
+    const auth = solidClientAuthentication;                                     
+    async function main(session){                                               
+        const kb = $rdf.graph();                                                
+        window.solidFetcher = session.fetch;                                    
+        const fetcher = $rdf.fetcher(kb)                                        
+        try {                                                                   
+          await fetcher.load(privateResource);                                  
+          alert("Private resource successfully loaded");                        
+        }                                                                       
+        catch(e) { alert(e) }                                                   
+    }                                                                           
+    document.getElementById('login').onclick = ()=> {                           
+        auth.login({                                                            
+            oidcIssuer: idp,                                                    
+            redirectUrl: window.location.href,                                  
+            clientName: "rdflib test"                                           
+        });                                                                     
+    }                                                                           
+    async function handleRedirectAfterLogin() {                                 
+        await auth.handleIncomingRedirect();                                    
+        session = auth.getDefaultSession();                                     
+        if (session.info.isLoggedIn)  main(session);                            
+    }                                                                           
+    handleRedirectAfterLogin();                                                 
+</script></html>                                                                
+```
+
+
+##  Custom fetchers
+
+Another way to specify an authenticated or alternate fetch is to do it when you create the fetcher.  This method follows the same steps as the first method with two exceptions : omit the line *global.solidFetch = auth.fetch* and instead of the line *const fetcher = $rdf.fetcher(kb)* use :
```

Comment:
```suggestion
Another way to specify an authenticated or alternate fetch is to do it when you create the fetcher.  This method follows the same steps as the first method, with two exceptions: omit the line *`global.solidFetch = auth.fetch`*; and instead of the line *`const fetcher = $rdf.fetcher(kb)`* use :
```